api4/emoji_test.go

```diff
@@ -33,6 +36,20 @@ func TestCreateEmoji(t *testing.T) {
 		th.RestoreDefaultRolePermissions(defaultRolePermissions)
 	}()
 
+	emojiWidth := app.MaxEmojiWidth * 2
+	emojiHeight := app.MaxEmojiHeight * 2
+	// check that emoji gets resized and is of expected type
+	checkEmojiFile := func(id, expectedImageType string) {
+		path, _ := fileutils.FindDir("data")
+		file, fileErr := os.Open(filepath.Join(path, "/emoji/"+id+"/image"))
+		require.NoError(t, fileErr)
```

Comment:
We want to close the file when we are done with it. I'd suggest adding a `defer file.Close()` right after the error check.

Reply:
done